### ISSUE TITLE
Can't access to translations on backoffice

### ISSUE THREAD

**Issue by caotico25** [PrestaShop/PrestaShop#12184]
**Describe the bug**
Since i updated to PS1.7.5, i can't access to translations. Error 500 is given. If i activate debug i get this:

![captura de pantalla 2019-01-16 a las 19 23 24](https://user-images.githubusercontent.com/6215402/51270047-439a9780-19c4-11e9-8220-5c6c8fbeda6e.png)

If I go to that folder, the required file does not exist

**Additionnal information**
PrestaShop version: 1.7.5
PHP version: 7.2

**Comment by khouloudbelguith** [comment]
Hi @caotico25,

Did you have any errors during the upgrade?
Can you please provide us the log of your upgrade? It can be found in the log.txt file, available in <admin folder>/autoupgrade/tmp/log.txt
Thanks!

**Comment by caotico25** [comment]
hi @khouloudbelguith,

I did not have any error during the upgrade, everything was supposedly fine.

This is my log.txt file:

[log.txt](https://github.com/PrestaShop/PrestaShop/files/2767995/log.txt)

**Comment by khouloudbelguith** [comment]
Hi @caotico25,

In your log you have warnings about some modules: `autosupplierorders`, `gmerchantcenter`, they could be the problem.
Thanks to disable those modules, clear cache, check & feedback.
Thanks!

**Comment by caotico25** [comment]
Hi @khouloudbelguith,
I checked this modules and they are working correctly despite those warnings.
In the module autoupplierorders are produced by an included library whose purpose is to treat excel files. It is third-party and works correctly even if it causes these warnings (constants that already exist).
The gmerchantcenter module has not been modified by us and those warnings come from the factory. During its execution and its execution of tasks it does not cause any error.
Also, these errors are not related in any way to the error by which I can not access the translations due to the lack of tpl files ...

I downloaded a new zip of prestashop 1.7.5 and it doesn't content the file mail.tpl of translation controller on it...

**Comment by khouloudbelguith** [comment]
@caotico25, the error says that you are missing a file, it could be some third party module missing a file.
I did not manage to reproduce the issue after upgrade from PS1.7.4.4 to PS1.7.5.0 using the 1-click upgrade v4.5.1.
I tried also with PS1.7.5.0.0 after upgrade from 1.7.0.0 & from PS1.7.4.2, and they are all OK.
The translation page is well displayed.
Thanks!

**Comment by caotico25** [comment]
Good morning @khouloudbelguith
Yesterday I made a new clean installation of prestashop 1.7.5 and after searching and comparing files I discovered the error.
It was an override the cause ... (I'm a bit embarrassed ...) The thing is that I tried to disable all overrides and non-native modules and clean the caches and the error persisted, so I focused on the error that was still showing the debug mode ...
Sorry I made you lose time with this when it was not your failure ... This was the first time I posted something, such was my level of despair, as I always look before in the last corner of the forums or research on my own.

Thank you very much for everything equally, you do a great job. Have a nice day!
